crt_portal/cts_forms/locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -1189,6 +1189,10 @@ msgstr "您的记录号码是："
 msgid "Print report"
 msgstr "打印报告"
 
+#: templates/forms/confirmation.html:62
+msgid "Save report"
+msgstr "了解您的权利"
```

Comment:
Should be "save report"?

<img width="1242" alt="image" src="https://github.com/usdoj-crt/crt-portal/assets/15126660/fb4bd108-0e94-4a66-bda1-50c7ea436dd1">